management-server/src/main/kotlin/ut/isep/management/service/converter/applicant/ApplicantUpdateConverter.kt

```diff
@@ -0,0 +1,18 @@
+package ut.isep.management.service.converter.applicant
+
+import dto.applicant.ApplicantUpdateDTO
+import org.springframework.stereotype.Component
+import ut.isep.management.model.entity.Applicant
+import ut.isep.management.service.converter.UpdateConverter
+
+@Component
+class ApplicantUpdateConverter : UpdateConverter<Applicant, ApplicantUpdateDTO> {
+    override fun updateEntity(entity: Applicant, updateDTO: ApplicantUpdateDTO): Applicant {
+        return entity.apply {
+            updateDTO.name?.let { this.name = it }
+            updateDTO.status?.let { this.status = it }
```

Comment:
same here with email

Reply:
Addressed in https://github.com/Alakind/ISEP/commit/51a8267b109ae3656dd1994b8a210bab7395b205